.github/workflows/run-tests.yml

```diff
@@ -6,72 +6,40 @@ on:
   pull_request:
     branches: [ main ]
 
-jobs:
-  build:
 
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-        # Test the oldest and newest configuration on Mac
-        - os: macos-latest
-          python-version: 3.8
-          toxenv: py38
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: macos-latest
-          python-version: '3.10'
-          toxenv: py310
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        # Test the oldest and newest configuration on Windows
-        - os: windows-latest
-          python-version: 3.8
-          toxenv: py38
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: windows-latest
+jobs:
+  test:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
+    with:
+      posargs: '--dbase-download-dir ~/.chianti'
+      toxdeps: "'tox<4' tox-pypi-filter"
+      envs: |
+        - macos: py38
+        - macos: py310
+        - windows: py38
+        - windows: py310
+        - linux: py38
+        - linux: py39
+        - linux: py310
+        - linux: build_docs
+          posargs: ''
           python-version: '3.10'
-          toxenv: py310
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        # Test all configurations on Linux
-        - os: ubuntu-latest
-          python-version: 3.8
-          toxenv: py38
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: ubuntu-latest
-          python-version: 3.9
-          toxenv: py39
-          cache_cmd: -- --dbase-download-dir ~/.chianti
-        - os: ubuntu-latest
+        - linux: pre-commit
+          posargs: ''
+      coverage: 'codecov'
+      cache-path: ~/.chianti
+      cache-key: chianti-${{ github.run_id }}
```

Comment:
What scope is this caching over?

Reply:
Every env can use the same database  so I don't want to download for every env.

My old cache key was this: https://github.com/wtbarnes/fiasco/blob/main/.github/workflows/run-tests.yml#L67. But I wasn't sure how to reproduce that here.